demos/simple_pipeline/demo_setup/README.md

```diff
@@ -0,0 +1,183 @@
+# Kubeflow demo - Simple pipeline
+
+This repository contains a demonstration of Kubeflow capabilities, suitable for
+presentation to public audiences.
+
+The base demo includes the following steps:
+
+1. [Setup your environment](#1-setup-your-environment)
+1. [Create a GKE cluster and install Kubeflow](#2-create-a-gke-cluster-and-install-kubeflow)
+1. [Install pipelines on GKE](#3-install-pipelines-on-gke)
+
+## 1. Setup your environment
+
+Set environment variables by sourcing the env file:
+
+```
+. kubeflow-demo-simple-pipeline.env
+```
+
+Create a clean python environment for installing Kubeflow Pipelines:
+
+```
+conda create --name kfp python=3.6
+source activate kfp
+```
+
+Install the Kubeflow Pipelines SDK:
+
+```
+pip install https://storage.googleapis.com/ml-pipeline/release/0.0.26/kfp-0.0.26.tar.gz --upgrade
+```
+
+## 2. Create a GKE cluster and install Kubeflow
```

Comment:
I guess there's a convenience to the user vs. maintainability tradeoff here. It's more convenient for the docs for launching kubeflow to be right here but it presents a maintainability challenge to have that documentation replicated in numerous places instead of being centralized. Thoughts?

Reply:
Sure it's your preference on that then.